Trappist/src/Promact.Trappist.Web/wwwroot/app/questions/category.model.ts

```diff
@@ -0,0 +1,7 @@
+﻿export class Category
+{
+    categoryName: string;
+    id: number;
+    createdDateTime: Date;
```

Comment:
no need to take createdDateTime and updatedDateTime at client side. updateddatetime spelling is also wrong.